### PR TITLE
refactor: normalize Angular and Angular.js sections/resources

### PR DESCRIPTION
**Pull request by davorpa**
## What does this PR do?
Add resource(s) | Add info | Improve repo

### Why is this valuable (or not)?

Completes to apply #4940 to all listings

Some usefull resources are been added to prevent single items section hierachy in listings where was mixed. Also sets creators, platforms fileformats... in those sections Angular.js, Typescript>Angular.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates. Completes #4940
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

### Review Tasks
- [x] Move resources. Angular :arrow_forward: Typescript - Angular, AngularJS :arrow_forward: Javascript - AngularJS
- [x] Normalize title AngularJS
- [ ] Add see also/crosslink notes

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: books/free-programming-books-et.md [hunk 1]
```diff
@@ -35,9 +35,9 @@
 * [Javascript](http://lepo.it.da.ut.ee/~epuman/javascript.html) - Ella Puman, Tartu ÜLikooli õppematerjal
 
 
-##### Angular
+##### Angular.Js
```
Comment (eshellman) [inline books/free-programming-books-et.md:38]: capital 'J'?

**Comment by eshellman** [comment]
If it's named at the source as "AngularJS" then why "angular.js"

Would it make sense to add "see also" links  between the two angular sections? Will someone looking for an Angular book to learn from know that it's not in the AngularJS section?

**Comment by davorpa** [comment]
> If it's named at the source as "AngularJS" then why "angular.js"
> 
> Would it make sense to add "see also" links between the two angular sections? Will someone looking for an Angular book to learn from know that it's not in the AngularJS section?

**Comment by davorpa** [comment]
> If it's named at the source as "AngularJS" then why "angular.js"

In English listing was typed as is. To be clear... You suggest revert headings and use AngularJS?

> Would it make sense to add "see also" links between the two angular sections? Will someone looking for an Angular book to learn from know that it's not in the AngularJS section?

Do you are talking to put a note at TOC entries? or how to abord? I don't know if linter accept it.

File: books/free-programming-books-et.md (hunk 1 above)
Comment (davorpa) [reply books/free-programming-books-et.md:38]: Yep. I misktake.

Thanks for your eyes :hugs:

**Comment by eshellman** [comment]
I think if the crosslink starts with … it will sort to the end of the list except for CJK …
HORIZONTAL ELLIPSIS
Unicode: U+2026, UTF-8: E2 80 A6
need to experiment

**Comment by eshellman** [comment]
Our general rule for titles, names, etc. is to use the form used by the creator. So based on https://angularjs.org/ , I would say we need to use AngularJS.

**Comment by eshellman** [comment]
so far so good

**Comment by davorpa** [comment]
> so far so good

Yep. I'm in progress yet :construction:. It was very weird pass linter this time, so I prefer force-amend instead sync :laughing:

Sorry for actions report spam. Altough, I have enabled the actions enviroment in my fork, I'm trying to have a remark/linter local enviroment with same fpb rules to avoid this kind of issues :wink: If it serve, I can pull it too as another PR. :wink:

Reviewing now about "see also" notes :rocket:

**Comment by davorpa** [comment]
> I think if the crosslink starts with … it will sort to the end of the list except for CJK …
> HORIZONTAL ELLIPSIS
> Unicode: U+2026, UTF-8: E2 80 A6
> need to experiment

I bring good news. No problems with linter :rocket; as first draft
I opt with quote syntax placed under section heading, taking care about [whitespaces rule 1-0-2](https://github.com/vhf/remark-lint-blank-lines-1-0-2). Ellipsis is also used as you suggest.

```markdown
* [Official Free Online Guide for Apache Wicket framework](http://wicket.apache.org/learn/#guide)


### JavaScript

> :notebook::information_source: See also &#8230; [TypeScript](#typescript)

* [Basic JavaScript for the impatient programmer](http://www.2ality.com/2013/06/basic-javascript.html) - Dr. Axel Rauschmayer (HTML)
```

Check the live preview deployed in my repo ([in english books](https://davorpa.github.io/free-programming-books/books/free-programming-books.html#javascript) or [in spanish books](https://davorpa.github.io/free-programming-books/books/free-programming-books-es.html#javascript)) before continue with other files and push changes:

image
&#8230;
image


Also let me know where crosslinks should we add (e.g. also in TOCs, in both directions like you can see now...) to have everything well tied :wink:.

**Comment by eshellman** [comment]
the blue "i" is a good idea - 2 icons is overkill

**Comment by davorpa** [comment]
> the blue "i" is a good idea - 2 icons is overkill

Maybe `:twisted_rightwards_arrows:` :twisted_rightwards_arrows: reffers better to crosslinks?

**Comment by eshellman** [comment]
> > the blue "i" is a good idea - 2 icons is overkill
> 
> Maybe `:twisted_rightwards_arrows:` 🔀 refers better to crosslinks?

the 'i' seems more versatile and easier to understand (for me at least)

**Review by eshellman — CHANGES_REQUESTED** [review]
As the community may have suggestions/ comments about the form and usage guidelines for crosslinks, I suggest that this PR should only use crosslinks for the Angular situation. Other crosslinks should be moved to a separate PR

**Comment by davorpa** [comment]
> As the community may have suggestions/ comments about the form and usage guidelines for crosslinks, I suggest that this PR should only use crosslinks for the Angular situation. Other crosslinks should be moved to a separate PR

Sorry for overscope inconvenient. Check it again

**Comment by eshellman** [comment]
OK, any more changes can go in a new pr.

**Pull request merged**